R/recipe.R

```diff
@@ -265,9 +276,11 @@ inline_check <- function(x) {
   funs <- funs[!(funs %in% c("~", "+", "-"))]
 
   if (length(funs) > 0) {
-    rlang::abort(paste0(
-      "No in-line functions should be used here; ",
-      "use steps to define baking actions."
+    cli::cli_abort(c(
+      x = "No in-line functions should be used here.",
+      i = "{cli::qty(length(funs))}The following function{?s} were found:",
```

Comment:
This should read "functions were" or "function was"---was/were needs to be variable as well.

Reply:
fixed in [fix pluralization](https://github.com/tidymodels/recipes/pull/1251/commits/59bcbb0886f8d247102a5dea87fc390ea55ae9ca)